README.md

```diff
@@ -1,3 +1,607 @@
-# @octokit/webhooks.js
+# @octokit/webhooks
 
-> GitHub webhooks toolset
+> GitHub webhooks toolset for Node.js
+
+[GitHub webhooks](https://developer.github.com/webhooks/) can be registered in multiple ways
+
+1. In repository or organization settings on [github.com](https://github.com/).
+2. Using the REST API for [repositories](https://developer.github.com/v3/repos/hooks/) or [organizations](https://developer.github.com/v3/orgs/hooks/)
+3. By installing a [GitHub App](https://developer.github.com/apps/).
+
+`@octokit/webhooks` helps you handle the webhooks sent by GitHub to your server.
+
+## Example
+
+```js
+// install with: npm install @octokit/webhooks
+const Webhooks = require('@octokit/webhooks')
+const receiver = Webhooks.receiver({secret: 'mysecret'})
+
+// called with installation webhook payload
+// see https://developer.github.com/v3/activity/events/types/#installationevent
+receiver.on('installation.deleted', {installation, sender} => {
+  console.log(`${sender.login} deleted installation for ${installation.account.login}`)
+  // logs e.g. "octocat deleted installation for octocat"
+})
+
+// called with installation event payload, same as above.
+// Hook handlers can return a promise or throw errors
+receiver.hook('installation.deleted', {installation} => {
+  return myApp.installations.remove(installation)
+})
+
+// called with installation event payload, same as above
+receiver.hook('installation.created', async {installation, sender} => {
+  const numAvailableInstallations = await myApp.installations.availableFor(sender.id)
+  if (numAvailableInstallations === 0) {
+    throw Error(`${sender.id} is out of installations`)
+  }
+
+  return myApp.installations.add(installation)
+})
+
+// pass in the webhook name, request payload and request signature and handle
+// errors that might have been thrown in hook handlers
+receiver.handle({
+  name: 'installation',
+  data: eventPayload,
+  signature
+}).catch(handleErrorsFromHooks)
```

Comment:
I think the most common use case will be people using the webhook handler as middleware, so my preference would be to have this first example be a version of the middleware example below.